closure/repositories.bzl

```diff
@@ -1073,6 +1076,58 @@ def rules_python():
         ],
     )
 
+def rules_webtesting():
+    http_archive(
+        name = "io_bazel_rules_go",
+        sha256 = "099a9fb96a376ccbbb7d291ed4ecbdfd42f6bc822ab77ae6f1b5cb9e914e94fa",
+        urls = [
+            "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.35.0/rules_go-v0.35.0.zip",
+            "https://github.com/bazelbuild/rules_go/releases/download/v0.35.0/rules_go-v0.35.0.zip",
+        ],
+    )
+
+    http_archive(
+        name = "bazel_gazelle",
+        sha256 = "62ca106be173579c0a167deb23358fdfe71ffa1e4cfdddf5582af26520f1c66f",
+        urls = [
+            "https://mirror.bazel.build/github.com/bazelbuild/bazel-gazelle/releases/download/v0.23.0/bazel-gazelle-v0.23.0.tar.gz",
+            "https://github.com/bazelbuild/bazel-gazelle/releases/download/v0.23.0/bazel-gazelle-v0.23.0.tar.gz",
+        ],
+    )
+
+    # TODO: This downloads the rules_webtesting repository by pinning to a commit.
+    # Update to download when rules_webtesting has a newer release (>=0.3.6).
+    # The 2 targets above: 'io_bazel_rules_go' and 'bazel_gazelle' will be included in the release,
+    # please delete the downloading of the targets and their use in //closure/testing/web_test_repositories.bzl.
```

Comment:
# TODO: Please remove the two following dependencies when rules_webtesting is pinned to an official release.